core/src/main/scala/org/apache/spark/ui/SparkUI.scala

```diff
@@ -100,6 +101,13 @@ private[spark] class SparkUI private (
     attachTab(stagesTab)
     attachTab(new StorageTab(this, store))
     attachTab(new EnvironmentTab(this, store))
+    if (sc.map(_.conf.get(DRIVER_LOG_LOCAL_DIR).nonEmpty).getOrElse(false)) {
+      val driverLogTab = new DriverLogTab(this)
+      attachTab(driverLogTab)
+      attachHandler(createServletHandler("/log",
```

Comment:
driver_log?

Reply:
It's used as the REST API prefix, `https://driver/log` and it should be `log` in order to reuse the existing `log-view.js` Javascript of Worker UIs.

https://github.com/apache/spark/blob/8d4ca0aa580f9622eb6f74bb99ae8a3e4e9667d5/core/src/main/resources/org/apache/spark/ui/static/log-view.js#L57-L66